src/cpp/database/database.cpp

```diff
@@ -3163,7 +3133,19 @@ void Database::check_entity_contains_all_references(
         check_entity_container_contains_id(reference_container, referenced_id);
```

Comment:
Now this method is only used in this context, i suggest delete the method and make the functionality here